src/common/util/src/file_util.cpp

```diff
@@ -348,17 +353,45 @@ std::wstring ov::util::string_to_wstring(const std::string& string) {
 }
 #endif
 
-std::string ov::util::get_absolute_file_path(const std::string& path) {
+std::string ov::util::get_absolute_file_path(const std::string& path, bool strict) {
     std::string absolutePath;
     absolutePath.resize(MAX_ABS_PATH);
-    auto absPath = get_absolute_path(&absolutePath[0], path);
-    if (!absPath) {
-        std::stringstream ss;
-        ss << "Can't get absolute file path for [" << path << "], err = " << strerror(errno);
-        throw std::runtime_error(ss.str());
+    if (strict) {
+        char* absPath;
+        // canonicalize path and check file existence and accessibility
+        absPath = get_absolute_path(&absolutePath[0], path);
```

Comment:
Use `get_absolute_path`, even if `absPath` is NULL (file doesn't exist, on Linux), `absolutePath` will contain concatenated path